### PR TITLE
Tension within anti-profiling designs

### PR DESCRIPTION
**Pull request by jyasskin**
***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/158.html#example-preventing-profiling" title="Last updated on Jan 4, 2023, 5:59 PM UTC (fe3db6f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/158/0b159b4...jyasskin:fe3db6f.html" title="Last updated on Jan 4, 2023, 5:59 PM UTC (fe3db6f)">Diff</a>

### CODE REVIEW

**Comment by yoavweiss** [comment]
Should this also include the possibility of applications moving off of the web and onto other application platforms that are more privacy invasive? Or does this belong in some other document?

**Comment by npdoty** [comment]
Is this an example of recommendations from different principles that are in tension? I don't know what the two different groups of principles are that the text refers to.

It seems more like an example of how mitigations are imperfect or how harm reduction can be used to mitigate but not eliminate privacy impacts.

**Comment by npdoty** [comment]
@jyasskin the updated text helps me understand the suggestion, thank you!

I don't particularly see recommendations being in tension here -- user agents should help users select the identity they want to present and should share the data that the user wants to share (and not the data they don't want to share). Maybe this will depend in part on how we nail down the principle about what data about a user is shared (e.g. #150).

I don't know that providing alternative targeting mechanisms is done for the purpose of, or is likely to have the effect, of largely decreasing the funding available to develop alternative tracking mechanisms, unless this alternative targeting mechanism is so effective and so widely used that additional surveillance could not have any marginal effect for any party -- a very high bar.

**Comment by npdoty** [comment]
I agree that "moving off the web" is a potential risk to consider. I'm not sure how exactly that would lead us to different conclusions for privacy on the Web (which is what I would consider the scope of the document), but I'd be open to pointers or thinking about that.

I'm certainly interested in ways to advocate and design for better privacy on other platforms as well. I think it's unlikely that an effective way to improve privacy on other platforms is to make sure we don't have too much privacy for users of the Web. :)

**Comment by jyasskin** [comment]
@wseltzer We'd agreed to kick tradeoffs between privacy and non-privacy concerns out to another document, but this one is between two different privacy goals, and I don't see another document that it would fit in. It's true that this is less direct than the other tradeoffs, but it's really important to consider and plan for these second-order effects when changing how the Web works.

**Comment by wseltzer** [comment]
@jyasskin I agree second-order effects are important, but I'm less convinced we can predict them confidently, nor that we can then enumerate subsequent follow-on effects including unintended consequences. I think it's a difference between general principle and implementation detail, where successful implementation depends on iteration and empirical testing.

**Comment by torgo** [comment]
cross-referencing #106

File: index.html [hunk 1]
```diff
@@ -837,6 +843,26 @@
...
+[[[#identity]]] by blocking this profiling, but because the profiles are valuable, there's
+a large incentive to work around user agent measures, sometimes by using techniques that
+are very expensive or impossible to block. If user agent behavior causes websites to adopt
+these alternate tracking methods, the web as a whole won't respect the [[[#identity]]]
+principles.
+
+User agents can reduce the incentive to develop these alternate tracking methods by
+building APIs to facilitate the most common and least harmful uses of user profiles.
+However, those APIs usually still reveal some information about a user's behavior. For
+example, [[[IPA]]] reveals a limited amount of information about each user in order to
```
Comment (jyasskin) [inline index.html:859]:
```suggestion
example, even the most privacy-respecting conversion attribution API will reveal a limited amount of information about each user in order to
```

**Comment by jyasskin** [comment]
Merging as discussed on [Jan 4](https://github.com/w3ctag/privacy-principles/blob/main/meetings/2023-01-04-minutes.md#pr-tension-within-anti-profiling-designs).

**Pull request merged**